base/applications/network/dwnl/dwnl.c

```diff
@@ -73,7 +73,8 @@ CBindStatusCallback_UpdateProgress(CBindStatusCallback *This)
         UINT Percentage;
 
         Percentage = (UINT)((This->Progress * 100) / This->Size);
-        if ((Percentage > 99) && (This->Progress != This->Size)) // If percentage is greater than 99% but sizes don't match, do a failsafe
+    // If percentage is greater than 99% but sizes don't match, do a failsafe.
+    if ((Percentage > 99) && (This->Progress != This->Size))
```

Comment:
Please fix the indentation.

Reply:
Addressed: f5eece6f9caba3f50583eab5d026efe7cb8da9f4 (plus: f4edd92ae18074e6395e8fda84fcf64dd53b023e )